CMakeLists.txt

```diff
@@ -70,5 +73,11 @@ if(IDF_TARGET STREQUAL "esp32" OR IDF_TARGET STREQUAL "esp32s2" OR IDF_TARGET ST
     message(WARNING "ESP-IDF version detected: '${idf_version}'.")
   endif()
 
-  register_component()
 endif()
+
+  set(COMPONENT_ADD_INCLUDEDIRS
```

Comment:
please move this block to the top and make sure that indentation is correct

Reply:
Hi @me-no-dev, this is now taken care of!